apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1470,10 +1474,23 @@ naming_system_manage_name(_Config) ->
     {ok, []} = rpc(aec_tx_pool, peek, [infinity]),
 
     %% Check that TTL and pointers got updated in name entry
-    TmpPointers = <<"{\"account_pubkey\":\"srtrst\"}">>,
     {ok, 200, #{<<"name">>     := Name,
                 <<"name_ttl">> := NameTTL,
-                <<"pointers">> := TmpPointers}} = get_name(Name),
+                <<"pointers">> := Pointers}} = get_name(Name),
+
+    {ok, 200, #{<<"balance">> := Balance3}} = get_balance(),
+    Host = internal_address(),
+    http_request(Host, post, "spend-tx",
+        #{recipient_pubkey => Name,
```

Comment:
I see this is the main point of the test.

Reply:
yes, correct